### DISCUSSION TITLE
Is there a way to change the DMG icon/background?

### DISCUSSION THREAD

**Discussion by wneessen** [goreleaser/goreleaser#4749] [Q&A] [answered]
I am looking for a way to change the icon of the DMG and (if possible the background picture of the DMG, once you open the folder). Is this possible with Goreleaser?

**Comment by caarlos0** [comment, marked as answer]
not yet, but should be doable! will add to the roadmap!

thanks!

**Reply by wneessen** [reply]
That's great! Appreciate the quick response and looking forward for this feature!
